src/components/Progress/Progress.tsx

```diff
@@ -10,7 +11,7 @@ export type ProgressTheme = 'default' | 'success' | 'warning' | 'danger' | 'info
 export type ProgressView = 'normal' | 'thin' | 'thinnest';
 export type ProgressValue = number;
 
-interface Stack {
+export interface Stack {
```

Comment:
If it's exported from the package it should be name like `ProgressStack` to avoid potential collision with other components